hw5/TableOfValues.java

```diff
@@ -0,0 +1,18 @@
+package hw4;
+
+/* 4. Написати програму, яка друкує таблицю значень sin(x),
+        де х - від 0 до 360 з кроком 10.(значення мають бути записані в масив за допомогою циклу)*/
+public class TableOfValues {
+
+    static void tableOfValues() {
+        double[] array = new double[37];
+        int j = 0;
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] =  Math.sin(i);
```

Comment:
So, in our array will be next values sin(0), sin(1), sin(2)..... sin(36)
Pls, fill array like sin(0), sin(10), sin(20)...sin(360)

Reply:
changed Math.sin(i) to Math.sin(j)